module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+constexpr uint32_t height = 32; //160;
```

Comment:
These comments are probably not very helpful - even misleading. is `160` some converted value from 32?

Reply:
+1